config/initializers/sufia.rb

```diff
@@ -208,7 +208,8 @@
      "2014-06" => "2014-06",
      "2014-11" => "2014-11",
      "2015-06" => "2015-06",
-     "2015-11" => "2015-11"
+     "2015-11" => "2015-11",
+     "2016-06" => "2016-06"
```

Comment:
Does it make sense to create a PR with this one line change? This has to be in, in order for admins to edit the newly migrated objects.